### ISSUE TITLE
New-HPOVLicense stopped working with 3.1 library

### ISSUE THREAD

**Issue by daveolker** [HewlettPackard/POSH-HPOneView#282]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior

New-HPOVLicense -File $License_File 

This should read the provided license file and import the licenses.  This worked fine with the 3.0 library.

### Actual Behavior
```powershell
New-HPOVLicense : The "type" field content, "License", included in this request is invalid. Correct the content in the 'type' field of the JSON and try the request again. 
+         New-HPOVLicense -File $License_File
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidResult: (LicenseKey:String) [New-HPOVLicense], LicenseKeyException
    + FullyQualifiedErrorId : InvalidResult,New-HPOVLicense
```
### Steps to reproduce

I've tried calling New-HPOVLicense in both modes - providing a text file with the licenses as well as calling it with -LicenseKey and passing in the licenses as an ASCII string.  Both methods work fine with the 3.0 library using the same license.txt file or licenses string.  Both fail with the 3.1 library.

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`):

PowerShellLibraryVersion : 3.10.1377.2109
LibraryPath              : C:\Program Files\WindowsPowerShell\Modules\HPOneView.310

HPE OneView Appliance Version:  3.10.05-0302465

Output from `$PSVersionTable` on your Windows Host:

Name                           Value                                                                                                                    
----                           -----                                                                                                                    
PSVersion                      5.1.14393.1480                                                                                                           
PSEdition                      Desktop                                                                                                                  
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                  
BuildVersion                   10.0.14393.1480                                                                                                          
CLRVersion                     4.0.30319.42000                                                                                                          
WSManStackVersion              3.0                                                                                                                      
PSRemotingProtocolVersion      2.3                                                                                                                      
SerializationVersion           1.1.0.1

**Comment by ChrisLynchHPE** [comment]
Thanks.  I found and fixed this issue, which will be in the next update.

**Comment by ChrisLynchHPE** [comment]
I have addressed this issue with in [Release 3.10.1416.3042](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.10.1416.3042).  Please confirm, and we can then close this issue.

**Comment by daveolker** [comment]
Thanks Chris.  I just tested the new library and it works perfectly.
